tests/snapshots_test.go

```diff
@@ -0,0 +1,358 @@
+package snapshots_test
+
+import (
+	"io"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/hexops/autogold/v2"
```

Comment:
https://github.com/hexops/autogold

Reply:
what is the suggestion?

Reply:
My bad, I copy pasted it via a comment to open the repository. I didn't intend to send it with my review 🤦‍♂️

I'm sorry